### DISCUSSION TITLE
今天在空洞被砸了555

### DISCUSSION THREAD

**Discussion by ZoriaYi** [share121/inter-knot#1788] [其他]
![1](https://github.com/user-attachments/assets/fb1bec23-4412-46cd-8409-449f3a88ddc7)
你说巧不巧，我刚打算从空洞撤离，一个箱子就砸到我头上了，怎么现在还有这么没素质的人！
于是我把箱子带回家了，刚刚拆开发现里面是一个黑盒子，有哪位认识这是什么吗

**Comment by ZoriaYi** [comment]
为什么我的图片发不出来

**Reply by ZoriaYi** [reply]
有了有了

**Comment by WORLD-executeME** [comment]
看标志是维多利亚家政的，你要不要去问问他们？
